readthedocs/restapi/permissions.py

```diff
@@ -46,6 +59,7 @@ def has_object_permission(self, request, view, obj):
 
 
 class APIRestrictedPermission(permissions.IsAdminUser):
```

Comment:
This class should not inherit from `IsAdminUser` as there is no relation at all between the two. `BasePermission` is a better base class for this.